layouts/partials/mail-subjects.html

```diff
@@ -0,0 +1,46 @@
+{{/* Return the five latest subjects from the mailing list.
+     Only checks the current and previous month. Duplicates are removed, when they follow each other directly. */}}
+
+{{ $current_month := now.Format "2006-January" }}
+{{ $current_month_url := printf "https://lists.berlin.freifunk.net/pipermail/berlin/%s/subject.html" $current_month }}
+{{ $current_month_response := resources.GetRemote $current_month_url }}
+{{ $current_month_html := $current_month_response.Content }}
+{{ $current_month_subjects := findRE "\\[Berlin-wireless\\] (.*?)\n" $current_month_html }}
+{{ $current_month_reversed_subjects := slice }}
+{{ $array_length := len $current_month_subjects }}
+{{- range $index, $element := $current_month_subjects -}}
+    {{ $current_month_reversed_subjects = $current_month_reversed_subjects | append (index $current_month_subjects (sub $array_length (add $index 1))) }}
+{{- end }}
+
+{{ $previous_month_date := now.AddDate 0 -1 0 }}
```

Comment:
Cant this just be loop going back in time until you find 5 subjects instead of having this hardcoded this way?